views/home.tmpl

```diff
@@ -245,7 +245,9 @@
                                 </span>
                             </div>
                             <div class="fs12 lh1rem text-black-50">
-                                <span data-target="homepage.targetPct">{{printf "%.2f" .PoolInfo.PercentTarget}}</span> % of target&nbsp;<span>{{intComma .PoolInfo.Target}}</span>
+                                <span data-target="homepage.targetPct">{{printf "%.2f" (toAbsValue $.PercentChange)}}%</span>
```

Comment:
The percent sign needs to be outside of the `data-target="homepage.targetPct"` span, otherwise it is lost when the homepage controller overwrites the value when new data is received by websocket. Here is what it looks like after a block comes in

![image](https://user-images.githubusercontent.com/6109680/58096689-4b995c80-7b9b-11e9-9335-2d76b9a71c12.png)


The percent sign is lost.

Reply:
on it

Reply:
this is done